debugger-launchjson.md

```diff
@@ -18,12 +18,16 @@ The `preLaunchTask` field runs the associated taskName in tasks.json before debu
 This will create a task that runs `dotnet build`. You can read more about tasks at [https://code.visualstudio.com/docs/editor/tasks](https://code.visualstudio.com/docs/editor/tasks).
 
 ## Program
-The program field is set to the path to the application dll or .NET Core host executable to launch. 
+The program field is set to the path of the application dll or .NET Core host executable to launch.
 
-Example: "${workspaceRoot}/bin/Debug/\<target-framework\>/\<project-name.dll\>" where:
+This property normally takes the form: "${workspaceRoot}/bin/Debug/\<target-framework\>/\<project-name.dll\>".
```

Comment:
FYI my thinking was this: is the default values - stick the `.dll` at the end since that is the formula that 98% of people should probably use, so hopefully it is the most clear way of telling people what they need to fill out. In the documentation, stick is as part of the name since there are cases for non-dll extensions.